vaadin-text-field-flow-parent/vaadin-text-field-flow-integration-tests/src/test/java/com/vaadin/flow/component/textfield/tests/validation/IntegerFieldValidationBasicIT.java

```diff
@@ -69,14 +69,14 @@ public void clientSideInvalidStateIsNotPropagatedToServer() {
     }
 
     @Test
-    public void triggerInputBlur_assertValidity() {
+    public void triggerBlur_assertValidity() {
```

Comment:
**note:** A leftover from https://github.com/vaadin/flow-components/pull/4336. Let me know if you feel it deserves a separate PR.

Reply:
After more consideration, I decided to create a separate PR containing this as well as a few other changes: https://github.com/vaadin/flow-components/pull/4342